content/en/docs/reference/glossary/cgroups.md

```diff
@@ -1,13 +1,13 @@
 ---
-title: CGroups
+title: CGroup
 id: cgroup
```

Comment:
nit: Should the title be `cgroup` instead of `CGroup` or `control group`?
I just checked the [v2 docs](https://www.kernel.org/doc/Documentation/cgroup-v2.txt):

"cgroup" stands for "control group" and is never capitalized.  The
singular form is used to designate the whole feature and also as a
qualifier as in "cgroup controllers".  When explicitly referring to
multiple individual control groups, the plural form "cgroups" is used.

cgroup is a mechanism to organize processes hierarchically and
distribute system resources along the hierarchy in a controlled and
configurable manner.